{"name": "Невозможность авторизоваться курьером с некорректным значением поля", "status": "passed", "steps": [{"name": "Отправка запроса на авторизацию курьера с некорректными данными", "status": "passed", "start": 1721243814207, "stop": 1721243814274}], "parameters": [{"name": "key", "value": "'login'"}, {"name": "wrong_field", "value": "'wrong@login'"}], "start": 1721243814206, "stop": 1721243814275, "uuid": "507c25ae-658f-4997-a1e5-c8b7ccfb72d4", "historyId": "c8214069ae2a0f58db953d0c4765a8f8", "testCaseId": "95bfda5ea90692b56bb543bf0b3a4bfb", "fullName": "tests.test_login_courier.TestLoginCourier#test_log_in_courier_with_wrong_field_failure", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-VR12VM9"}, {"name": "thread", "value": "13348-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}